Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -544,14 +596,20 @@ public void OnLoggedIn()
                 return;
             }
 
-            SubscribteToGameInvites();
+            SubscribeToGameInvites();
             SubscribeToSessionMessageConnectionRequests();
 
             CurrentInvite = null;
 
             userLoggedIn = true;
         }
 
+        /// <summary>
+        /// Unsubscribes from invitations and Peer2Peer message requests and clears internal states.
+        /// This can be run before or after a successful logout, or preparing to shut down the application.
+        /// This must be called in order to remove Native notifications.
```

Comment:
It may be wise to be clearer regarding what Native notifications means in this context.

Reply:
Sure, I think the easiest way to do that here is just to point at OnLoggedIn, so changed the comments for that.